mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -218,6 +218,19 @@ def handle_roi_clicked(self, roi: SpectrumROI) -> None:
             self.view.table_view.current_roi_name = roi.name
             self.view.table_view.last_clicked_roi = roi.name
             self.view.set_roi_properties()
+            self.view.set_selected_roi(roi.name)
+
+    def handle_roi_selection(self):
+        """ Handles user selection from the dropdown menu. """
+        selected_roi = self.view.roiSelectionWidget.roiDropdown.currentText()
+        if selected_roi:
+            self.view.current_roi_name = selected_roi
+            self.view.set_roi_properties()
+            self.redraw_spectrum(selected_roi)
```

Comment:
For now, we don't need to do anything when the selection is changed in the `ROISelectionWidget` (that will happen in the new widgets)